test/v1/services/AmendDisclosuresServiceSpec.scala

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.services
+
+import uk.gov.hmrc.domain.Nino
+import v1.controllers.EndpointLogContext
+import v1.mocks.connectors.MockAmendDisclosuresConnector
+import v1.models.domain.DesTaxYear
+import v1.models.errors._
+import v1.models.outcomes.ResponseWrapper
+import v1.models.request.disclosures.{AmendDisclosuresRequest, AmendDisclosuresRequestBody, AmendTaxAvoidance}
+
+import scala.concurrent.Future
+
+class AmendDisclosuresServiceSpec extends ServiceSpec {
+
+  private val nino = "AA112233A"
+  private val taxYear = "2019"
+  private val correlationId = "X-corr"
+
+  val amendTaxAvoidance: AmendTaxAvoidance = AmendTaxAvoidance("12345","2020-12")
+
+  val amendDisclosuresRequest: AmendDisclosuresRequest = AmendDisclosuresRequest(
+      nino = Nino(nino),
+      desTaxYear = DesTaxYear(taxYear),
```

Comment:
Can you please change desTaxYear field in the AmendDisclosuresRequest to taxYear and make appropriate changes in the connector, connector spec and so on.

Reply:
Is there a particular reason, as income-api uses it too? Also that DesTaxYear is already in master, you merged it last week. But I can change it as part of this story if needed.

Reply:
Not the DesTaxYear but the desTaxYear so I mean the field instead.

Reply:
Ah okay you mean the field name/case class parameter name, I'll make that change, thanks.

Reply:
Yep, thanks